ara/plugins/lookup/ara_api.py

```diff
@@ -1,14 +1,11 @@
 # Copyright (c) 2022 The ARA Records Ansible authors
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-from __future__ import absolute_import, division, print_function
 
 from ansible.plugins.lookup import LookupBase
 
 from ara.clients import utils as client_utils
 
-__metaclass__ = type
-
```

Comment:
These need to stay.

I don't make the rules: they're part of the Ansible module boilerplate i.e, https://github.com/ansible/ansible/blob/79f67ed56116be11b1c992fade04acf06d9208d1/lib/ansible/plugins/lookup/dict.py#L4-L5